Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -454,8 +455,9 @@ public class CameraManager {
     /**
      Return a new center coordinate shifted by a given offset value.
      - Parameter offset: The `CGPoint` value to shift the map's center by.
+     - Parameter drift: Indicates if the camera is experiencing a fling, which means the offset value needs to be manipulated
      */
-    func shiftCenterCoordinate(by offset: CGPoint) -> CLLocationCoordinate2D {
+    func shiftCenterCoordinate(by offset: CGPoint, drift: Bool) -> CLLocationCoordinate2D {
```

Comment:
This is inaccurate, isn't it? This `drift` property only does something if the map is pitched it seems like.

Reply:
correct, so if the shift is reflecting a drift, we need to account for pitch in the code. What would be a more appropriate fix here?